source/extensions/dynamic_modules/sdk/rust/src/lib.rs

```diff
@@ -489,6 +500,34 @@ impl EnvoyHttpFilter for EnvoyHttpFilterImpl {
       )
     }
   }
+
+  fn send_response(&mut self, status_code: u32, headers: Vec<(&str, &[u8])>, body: Option<&str>) {
+    let body_ptr = body.map(|s| s.as_ptr()).unwrap_or(std::ptr::null());
+    let body_length = body.map(|s| s.len()).unwrap_or(0);
+
+    let headers_c: Vec<envoy_dynamic_module_type_module_http_header> = headers
+      .iter()
+      .map(
+        |(key, value)| envoy_dynamic_module_type_module_http_header {
+          key_ptr: key.as_ptr() as *mut _,
+          key_length: key.len(),
+          value_ptr: value.as_ptr() as *mut _,
+          value_length: value.len(),
+        },
+      )
+      .collect();
```

Comment:
I _believe_ this is not needed - the memory representation `(&str, &[u8])` should be exactly the same as `envoy_dynamic_module_type_module_http_header` (that's why get_request_headers variant callbacks are working)

Reply:
looks like it did work!